docs/architecture/adr-063-privval-grpc.md

```diff
@@ -0,0 +1,75 @@
+# ADR 063: Privval gRPC
+
+## Changelog
+
+- 23/11/2020: Initial Version (@marbar3778)
+
+## Context
+
+Validators use remote signers to help secure their keys. This system is Tendermint's recommended way to secure validators, but the path to integration with Tendermint's private validator client is plagued with custom protocols. 
+
+Tendermint uses its own custom secure connection protocol (`SecretConnection`) and a raw tcp/unix socket connection protocol. The secure connection protocol until recently was exposed to man in the middle attacks and limits the amount and speed of integrations users can make. The raw tcp connection protocol is less custom, but has been causing minuet issues with users. 
+
+Migrating Tendermint's private validator to a widely adopted protocol will ease the current maintenance and integration burden experienced with the current protocol. 
+
+## Decision
+
+After discussing with multiple stake holders, [gRPC](https://grpc.io/) was decided on to replace the current private validator protocol. GRPC is a widely adopted protocol in the micro-service and cloud infrastructure world. GRPC uses [protocol-buffers](https://developers.google.com/protocol-buffers) to describe its services, providing a language agnostic implementation. Tendermint uses protobuf for on disk and over the wire encoding already making the integration with gRPC simpler. 
```

Comment:
just a nit but perhaps be consistent with calling it gRPC (sometimes you have GRPC)